src/lib/journey/callbacks/text-output/text-output.mock.ts

```diff
@@ -1,5 +1,67 @@
-export const docsExample = {
-  type: 'TextOutputCallback',
+import { CallbackType } from '@forgerock/javascript-sdk';
+
+export default {
+  authId: 'foo',
+  callbacks: [
+    {
+      type: CallbackType.TextOutputCallback,
```

Comment:
Yes, since `SuspendedTextOutputCallback` extends `TextOutputCalllback` with zero additional behavior, it's essentially a copy. The other variations of "suspend" were just a typo.

Rather than having identical components for the sake of matching callbacks with different names, I went with one callback to handle both. That way we don't have to keep two components in-sync if something changes.

Reply:
Makes sense to me - just wanted to check what we decided as there was a conversation on this further up. Thanks